app/services/bus_notice_service.py

```
@@ -49,10 +49,13 @@ def _log_task_error(task: asyncio.Task):
 
             cls._image_task = asyncio.create_task(cls.generate_all_route_images())
             cls._image_task.add_done_callback(_log_task_error)
+            return "success"
             
         except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)